src/04-date-tasks.js

```diff
@@ -53,10 +53,19 @@ function parseDataFromIso8601(/* value */) {
  *    Date(2012,1,1)    => true
  *    Date(2015,1,1)    => false
  */
-function isLeapYear(/* date */) {
-  throw new Error('Not implemented');
-}
+function isLeapYear(date) {
+  const YEAR = date.getFullYear();
+  let result = true;
+  if (YEAR % 4 !== 0) {
+    result = false;
+  } else if (YEAR % 100 !== 0) {
+    result = true;
+  } else if (YEAR % 400 !== 0) {
```

Comment:
"if ... else if .... else" conditions hard to read.
It would be great to have smth like: 
If () return smth
if () return anotherSmth
if () return smthElse